xml/geo_manage_i.xml

```diff
@@ -28,161 +28,152 @@
+     <literal>ticket-nfs</literal> to <literal>&cluster2;</literal>. To do so,
+    proceed as follows: </para>
+   <step>
+    <para> Log in to <literal>&cluster2;</literal>.</para>
+   </step>
    <step>
-    <para>
-     Set <literal>&ticket1;</literal> to standby with the following command:
-    </para>
-<screen>&prompt.root;<command>crm_ticket</command> -t &ticket1; -s</screen>
+    <para> Grant <literal>ticket-nfs</literal>to site
+      <literal>&cluster2;</literal> using the <option>-F</option> option: </para>
+    <screen>&prompt.root;<command>booth</command> grant -F ticket-nfs</screen>
+    <para>You will see a warning that the same ticket might be granted to
+     another site, but the command will be executed.</para>
    </step>
    <step>
-    <para>
-     Wait for any resources that depend on <literal>&ticket1;</literal> to be
-     stopped or demoted cleanly.
-    </para>
+    <para>Check the result with:</para>
+    <screen>&prompt.root;<command>booth</command> list</screen>
+    <para>It should show <literal>&cluster2;</literal> as ticket owner for
+      <literal>ticket-nfs</literal> now. All resources that depend on this
+     ticket will be started on <literal>&cluster2;</literal>.</para>
    </step>
    <step>
-    <para>
-     Revoke <literal>&ticket1;</literal> from site
-     <literal>&cluster1;</literal> with:
-    </para>
-<screen>&prompt.root;<command>booth</command> revoke -s 192.168.201.100 &ticket1;</screen>
+    <para> Before trying to get <literal>&cluster1;</literal> up and running
+     again, set <literal>ticket-nfs</literal> to standby on <literal>&cluster2;</literal> (to
+     prevent that resources will be started on both sites in parallel): </para>
+    <screen>&prompt.root;<command>crm_ticket</command> -t ticket-nfs --standby</screen>
    </step>
    <step>
-    <para>
-     After the ticket has been revoked from its original site, grant it to the
-     site <literal>&cluster2;</literal> with:
-    </para>
-<screen>&prompt.root;<command>booth</command> grant -s 192.168.202.100 &ticket1;</screen>
+    <para> Wait for any resources that depend on <literal>ticket-nfs</literal>
+     to be stopped or demoted cleanly. </para>
+   </step>
+   <step>
+    <para>Get <literal>&cluster1;</literal> up and running again. If you want
+     to keep the resources running on site <literal>&cluster2;</literal>,
+     revoke <literal>ticket-nfs</literal> from <literal>&cluster1;</literal>: </para>
+    <screen>&prompt.root;<command>booth</command> revoke -s 192.168.201.100 ticket-nfs</screen>
+   </step>
+   <step>
+    <para>Log in to site <literal>&cluster2;</literal> and put
+      <literal>ticket-nfs</literal> into active mode again:</para>
+    <screen>&prompt.root;<command>crm_ticket</command> -t ticket-nfs --activate</screen>
+    <para>All resources that depend on this ticket will be started on
+      <literal>&cluster2;</literal> again. </para>
```

Comment:
I can thinking of these potentially interesting management scenarios:

- "Manually move an automatic ticket":
1) put the ticket to standby from cluster1" 
2) "grant the ticket to cluster2" 
3) "activate ticket on cluster1 if we want the resources automatically fail over to cluster1 in case cluster2 fails "
 -- resources run only if the ticket is granted & non-standby.

- "Move a manual ticket"
1) "put the ticket to standby from cluster1"
2) "revoke the ticket from cluster1" 
  -- @Chriskowalczyk If I remember correctly, manual ticket should be explicitly revoked before being granted again, am I correct?
3) "grant the ticket to cluster2" 
4)"activate ticket on cluster1 if we want the resources automatically fail over to cluster1 in case cluster2 fails " 

- "Fail over a manual ticket"
The procedure we already have in here :)

Reply:
@gao-yan , you can grant manual ticket a second time, with -F flag